packages/client/tests/functional/decimal/precision/tests.ts

```diff
@@ -62,5 +62,9 @@ testMatrix.setupTestSuite(
         mongo - decimals are not supported
     `,
     },
+    skipEngine: {
+      from: ['wasm'],
+      reason: 'Fails on init with `unwrap_throw` failed',
+    },
```

Comment:
I think this is no longer the case, but the test gets stuck locally for me:

```
pnpm run test:functional --flavor js_neon --engine-type wasm --runInBand decimal/precision
```

Reply:
Trying out with `js_pg`, I confirm this problem is gone and we can uncomment the test.